src/component/SelectSearch.tsx

```diff
@@ -16,10 +16,11 @@ export interface SelectSearchProps {
     closeOnSelect?: boolean;
     initial?: SearchResult[];
     localSearchOptions?: SearchResult[];
-    _className?: string;
+    __className?: string;
     createPrompt?: string;
     onChange?: (selections: SearchResult | SearchResult[]) => void;
     isValidNewOption?: (option: { label: string }) => boolean;
+    isOptionUnique?: Function;
```

Comment:
Isn't this function a consistent signature?

Reply:
Yes that was lazy of me 😅. While addressing this I realized we could be leaning on `react-select` types a lot more that we do. What are your thoughts on https://github.com/nyaruka/floweditor/pull/252/commits/7d526f166f06300642a6c31ec9c6ae147d6ab817?